components/Cart/interfaces.rst

```diff
@@ -0,0 +1,102 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_cart_model_cart-interface:
+
+CartInterface
+~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single Cart.
+
+.. note::
+    This interface extends the :ref:`component_order_model_order-interface` |br|
+    For more detailed information go to `Sylius API's CartInterface`_.
+
+.. _Sylius API's CartInterface: http://api.sylius.org/Sylius/Component/Cart/Model/CartInterface.html
+
+.. _component_cart_model_cart-item-interface:
+
+CartItemInterface
+~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single CartItem.
+
+.. node::
+    This interface extends the :ref:`component_order_model_order-item-interface` |br|
+    For more detailed information go to `Sylius API's CartItemInterface`_.
+
+.. _Sylius API's CartItemInterface: http://api.sylius.org/Sylius/Component/Cart/Model/CartItemInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_cart_provider_cart-provider-interface:
+
+Cart provider
```

Comment:
`CartProviderInterface` - I suppose we should use full names in the headlines for all interfaces below.